pyqtgraph/graphicsItems/PlotItem/PlotItem.py

```diff
@@ -289,7 +273,31 @@ def __init__(self, parent=None, name=None, labels=None, title=None, viewBox=None
         if len(kargs) > 0:
             self.plot(**kargs)
         
-        
+    def setAxes(self, axisItems):
```

Comment:
if `self.axes` is actually non empty when this method is called, should not they be removed from the layout before adding the new axes?